assets/global.js

```diff
@@ -889,9 +896,12 @@ class VariantSelects extends HTMLElement {
     const addButton = button.querySelector('[name="add"]');
     const addButtonText = button.querySelector('[name="add"] > span');
     const price = document.getElementById(`price-${this.dataset.section}`);
+    const sku = document.getElementById(`Sku-${this.dataset.section}`);
+
     if (!addButton) return;
     addButtonText.textContent = window.variantStrings.unavailable;
     if (price) price.classList.add('visibility-hidden');
+    if (sku) sku.classList.add('visibility-hidden'), sku.setAttribute('aria-hidden', 'true');
```

Comment:
```suggestion
    if (sku) sku.classList.add('visibility-hidden'), sku.setAttribute('aria-hidden', 'true');
```
You should only need the `visibility: hidden` here.